core/src/test/scala/org/apache/spark/scheduler/WorkerDecommissionSuite.scala

```diff
@@ -31,7 +31,7 @@ import org.apache.spark.util.{RpcUtils, SerializableBuffer, ThreadUtils}
 class WorkerDecommissionSuite extends SparkFunSuite with LocalSparkContext {
 
   override def beforeEach(): Unit = {
-    val conf = new SparkConf().setAppName("test").setMaster("local")
```

Comment:
Is master = "local" the default ? And that's why the .setMaster("local") is removed ?

Reply:
No. See line 37 below. `local` is overridden by `local-cluster[2, 1, 1024]`. (I got confused when I was debugging the test failure...so I fixed it.)